pkg/deploytest/localgrpctransport.go

```diff
@@ -0,0 +1,37 @@
+package deploytest
+
+import (
+	"fmt"
+	"github.com/filecoin-project/mir/pkg/logging"
```

Comment:
unsorted

Reply:
It is sorted.
Note that it is done automatically by the IDE (which invokes gofmt).